docs/guides/cli/01_installation.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: installation
+title: Installation
+---
+
+## Overview
+
+Ory CLI is a convenient and easy-to-use tool that helps you manage and configure
+Ory Cloud and self-hosted projects.
+
+Its functionality is also essential for self-hosting Ory components and local
+development. The built-in proxy and tunneling mechanisms help route application
+traffic to the appropriate endpoints in a safe and hassle-free manner.
+
+The ability to export configurations allows you to bring your setup from a
+self-hosted environment to the cloud and from Ory Cloud to self-hosted setups
+alike.
+
+:::info
+
+The Ory CLI is actively developed. This means that new features are added with
+every release.
+
+:::
+
+## Installation
+
+You can install Ory CLI directly on your machine, or run the tool in Docker.
+
+### macOS
+
+Run this command to install the CLI using Homebrew:
+
+```shell
+brew install ory/tap/cli
+```
+
+Verify the installation:
+
+```shell
+ory help
+```
+
+### Linux
+
+Ory CLI works on all Linux distributions. Run this command to install:
+
+```shell
+bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -b . ory
+```
+
+You may want to move the Ory CLI to your `$PATH`:
+
+```shell
+sudo mv ./ory /usr/local/bin/
+```
+
+Verify the installation after moving to `$PATH`:
+
+```shell
+ory help
+```
+
+### Windows
+
+To install the CLI on Windows, use [scoop](https://scoop.sh). Run:
+
+```shell
+> scoop bucket add ory https://github.com/ory/scoop.git
+> scoop install ory
+```
+
+Verify the installation:
+
+```shell
+ory help
+```
+
+### Docker
+
+:::warning
+
+If you run Ory CLI in Docker, you can't Ory Proxy and Ory Tunnel.
+
+:::
+
+```shell
+### Pull Ory CLI image
+docker pull oryd/ory
+
+### Run image, enter container, and run a CLI command
+docker run --rm -it oryd/ory help
+```
+
+### Binary
+
+You can download the server and client binaries from the
+[GitHub "releases" tab](https://github.com/ory/cli/releases).
+
+To install, add the binary to the `PATH` environment variable or move it to a
+path that's already in your `$PATH`, for example `/usr/local/bin`.
```

Comment:
This is basically a copy of https://github.com/ory/docs/blob/master/docs/_common/install.mdx

Can we import that file instead? :)

Reply:
Thanks for pointing this out! 

No, I don't want to import this. This is a one-off document describing the installation steps for Ory CLI and there has to be a single source of truth for that. This content is not going to be re-used in any other document. If need be, we should link to it. 

What I'd rather do is look at that snippet you linked to and probably delete it. 

From my point of view, imports should be used sparsely, only when there's an actual need to have the exact same piece of content in multiple places.

Reply:
Just checked what that import is and it's for installing kratos/hydra/keto locally so doesn't really make sense to import the content you mentioned.